graph.go

```diff
@@ -448,7 +449,10 @@ func (g *Graph) Missing() []TransactionID {
 	g.RLock()
 	missing := make([]TransactionID, 0, len(g.missing))
 
-	for id := range g.missing {
+	for id, depth := range g.missing {
+		if g.rootDepth > conf.GetMaxDepthDiff()+depth {
```

Comment:
but how is this possible? such transactions should be deleted from missing on root depth update

Reply:
It's not clear -- it seems to have not been effective as we still got the error identified in #230 with this branch

Reply:
I am not sure yet. This is not an actual fix, just trying to see if doing this makes the error go away. Looks like the error is still there.